homeassistant/core.py

```diff
@@ -330,11 +330,49 @@ def _async_stop_handler(self, *args):
         self.exit_code = 0
         self.loop.create_task(self.async_stop())
 
+    @asyncio.coroutine
+    def _notify(self, message):
```

Comment:
There is no need for this function. Every component already exports functions for their services: https://github.com/andrey-git/home-assistant/blob/7b5b1fb7511bbc42c943d27eb0563a1e17ac4d24/homeassistant/components/persistent_notification.py#L47-L58